_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,209 @@
+- The specific presentation of the text rendered in the image resource can be achieved through formatted text. Otherwise, the rule might fail while [SC 1.4.5 Images of Text][sc1.4.5] and [SC 1.4.9 Images of Text (No Exception)][sc1.4.9] might by satisfied.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+This rule is designed specifically for [SC 1.4.5 Images of Text][sc1.4.5] which includes exceptions to the images it applies to. [SC 1.4.9 Images of Text (No Exception)][sc1.4.9] does not consider any exceptions. Therefore, some images that are inapplicable for this rule can be applicable to [SC 1.4.9 Images of Text (No Exception)][sc1.4.9].
+
+- [Understanding Success Criterion 1.4.5: Images of Text][sc1.4.5]
+- [Understanding Success Criterion 1.4.9: Images of Text (No Exception)][sc1.4.9]
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `img` element references an image resource that does not contain text.
+
+```html
+<img src="/test-assets/shared/fireworks.jpg" alt="fireworks going off behind the Eiffel tower at night" />
+```
+
+#### Passed Example 2
+
+This `input` element references an image resource that does not contain text.
+
+```html
+<input type="image" src="test-assets/shared/file.svg" alt="New file" />
+```
+
+#### Passed Example 3
+
+This `svg` element displays an image resource that does not contain text.
+
+```html
+<svg width="2in" height="3in" xmlns="http://www.w3.org/2000/svg">
+	<image x="20" y="20" width="200px" height="100px" href="/test-assets/shared/fireworks.jpg">
+		<title>Fireworks in Paris</title>
+	</image>
+</svg>
+```
+
+#### Passed Example 4
+
+This `object` element references an image resource where text is not the most significant content.
+
+```html
+<object data="/test-assets/0va7u6/times_square.jpg"></object>
+```
+
+#### Passed Example 5
+
+This `img` element references an image resource that contains text but where the presentation of the text is essential to convey the information.
+
+```html
+<p>The following image illustrates the use of cursive writing in the late nineteenth century.</p>
+
+<canvas id="canvas"> Your browser does not support the HTML5 canvas element. </canvas>
+
+<script>
+	var img = new Image()
+	img.src = '/test-assets/0va7u6/letter.jpg'
+	img.onload = function () {
+		fill_canvas(img)
+	}
+	function fill_canvas(img) {
+		var canvas = document.getElementById('canvas')
+		var ctx = canvas.getContext('2d')
+		canvas.width = img.width
+		canvas.height = img.height
+		ctx.drawImage(img, 0, 0)
+	}
+</script>
+```
+
+#### Passed Example 6
+
+This `div` element has a background image with a logo with text. Logotypes are considered an essential exception.
+
+```html
+<div
+	style="
+    width: 100px;
+    height: 100px;
+    background-image: url(test-assets/shared/w3c-logo.png);
+    background-repeat: no-repeat;
+  "
+></div>
+```
+
+#### Passed Example 7
+
+This `img` element is [purely decorative][].
```

Comment:
We should make this a failed example. An image doesn't become decorative when you put a caption under it. The information might be redundant, but it's still information. This is how we interpreted it in the decorative images rule. Specifically [inapplicable example 8](https://act-rules.github.io/rules/e88epe#inapplicable-example-8) has this case in it.

Reply:
I've updated the example (alt text and image used) so that this could be a passed example and moved this one to a failed example.

Reply:
This description needs an update.